sdk/src/main/java/com/hedera/hashgraph/sdk/RequestType.java

```diff
@@ -0,0 +1,436 @@
+        } else if(code == HederaFunctionality.CryptoAccountAutoRenew) {
+            return CRYPTO_ACCOUNT_AUTO_RENEW;
+        } else if(code == HederaFunctionality.ContractAutoRenew) {
+            return CONTRACT_AUTO_RENEW;
+        } else if(code == HederaFunctionality.GetVersionInfo) {
+            return GET_VERSION_INFO;
+        } else if(code == HederaFunctionality.TransactionGetReceipt) {
+            return TRANSACTION_GET_RECEIPT;
+        } else if(code == HederaFunctionality.ConsensusCreateTopic) {
+            return CONSENSUS_CREATE_TOPIC;
+        } else if(code == HederaFunctionality.ConsensusUpdateTopic) {
+            return CONSENSUS_UPDATE_TOPIC;
+        } else if(code == HederaFunctionality.ConsensusDeleteTopic) {
+            return CONSENSUS_DELETE_TOPIC;
+        } else if(code == HederaFunctionality.ConsensusGetTopicInfo) {
+            return CONSENSUS_GET_TOPIC_INFO;
+        } else if(code == HederaFunctionality.ConsensusSubmitMessage) {
+            return CONSENSUS_SUBMIT_MESSAGE;
+        } else if(code == HederaFunctionality.TokenCreate) {
+            return TOKEN_CREATE;
+        } else if(code == HederaFunctionality.TokenGetInfo) {
+            return TOKEN_GET_INFO;
+        } else if(code == HederaFunctionality.TokenFreezeAccount) {
+            return TOKEN_FREEZE_ACCOUNT;
+        } else if(code == HederaFunctionality.TokenUnfreezeAccount) {
+            return TOKEN_UNFREEZE_ACCOUNT;
+        } else if(code == HederaFunctionality.TokenGrantKycToAccount) {
+            return TOKEN_GRANT_KYC_TO_ACCOUNT;
+        } else if(code == HederaFunctionality.TokenRevokeKycFromAccount) {
+            return TOKEN_REVOKE_KYC_FROM_ACCOUNT;
+        } else if(code == HederaFunctionality.TokenDelete) {
+            return TOKEN_DELETE;
+        } else if(code == HederaFunctionality.TokenUpdate) {
+            return TOKEN_UPDATE;
+        } else if(code == HederaFunctionality.TokenMint) {
+            return TOKEN_MINT;
+        } else if(code == HederaFunctionality.TokenBurn) {
+            return TOKEN_BURN;
+        } else if(code == HederaFunctionality.TokenAccountWipe) {
+            return TOKEN_ACCOUNT_WIPE;
+        } else if(code == HederaFunctionality.TokenAssociateToAccount) {
+            return TOKEN_ASSOCIATE_TO_ACCOUNT;
+        } else if(code == HederaFunctionality.TokenDissociateFromAccount) {
+            return TOKEN_DISSOCIATE_FROM_ACCOUNT;
+        } else if(code == HederaFunctionality.ScheduleCreate) {
+            return SCHEDULE_CREATE;
+        } else if(code == HederaFunctionality.ScheduleDelete) {
+            return SCHEDULE_DELETE;
+        } else if(code == HederaFunctionality.ScheduleSign) {
+            return SCHEDULE_SIGN;
+        } else if(code == HederaFunctionality.ScheduleGetInfo) {
+            return SCHEDULE_GET_INFO;
+        } else {
+            return NONE;
+        }
+    }
+
+    @Override
+    public String toString() {
+        return code.name();
```

Comment:
As much as I want to use `code.name()` just line in `Status.java` I don't think we should. The reason is because `Status` contains a  `ResponseCodeEnum` which use proper UPPER_SNAKE_CASE for it's names, while `HederaFunctionaltiy` does not do this. We should print the value in UPPER_SNAKE_CASE here. This will probably involve another switch.

Reply:
Well, in this case at least the switch statement should actually work.  =P